src/server/services/migrateDB.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const { log } = require('../log');
+const Migration = require('../models/Migration');
+const { ask, terminateReadline } = require('./utils');
+const { findMaxSemanticVersion } = require('../util');
+const { showPossibleMigrations, migrateAll, getUniqueVersions } = require('../migrations/migrateDatabase');
+const migrationList = require('../migrations/registerMigration');
+
+function findMaxVersion(list) {
+	return findMaxSemanticVersion(getUniqueVersions(list));
+}
+
+(async () => {
+	let toVersion;
+	try {
+		const updateMax = await ask('Do you want to update to the max version? [yes/no]: ');
```

Comment:
My only wish is that this migratedb script could take in a version parameter argument (see createUser) that accepts a semantic version number or "highest" and does not go interactive. Thus, it could be run in the new `src/scripts/update.sh` script like `npm run migratedb --version highest`.